recipes/pangolin/meta.yaml

```diff
@@ -19,23 +19,23 @@ requirements:
     - pip
   run:
     - python >=3.7
-    - biopython ==1.74
+    - biopython >=1.74
     - pandas >=1.0.1
-    - tabulate
     - joblib >=0.11
-    - scikit-learn <1.3.0
+    - scikit-learn >=0.23.1
```

Comment:
This one is not helpful, see https://github.com/bioconda/bioconda-recipes/pull/42145/commits/ce574e73f5061c4f32416c2d341cbfcfcfcb543b

Reply:
So we should probably update upstream setup.py/environment.yml - instead of just fixing downstream :)

Thanks for sharing your insights!

Reply:
```suggestion
    - scikit-learn >=0.23.1,<1.3.0
```